modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/operation/DefaultExecutionMediator.java

```diff
@@ -89,13 +91,14 @@ public DefaultExecutionMediator(ExtensionModel extensionModel, OperationModel op
    * @throws Exception if the operation or a {@link Interceptor#before(ExecutionContext)} invokation fails
    */
   @Override
-  public Mono<Object> execute(OperationExecutor executor, ExecutionContextAdapter context) throws MuleException {
+  public Publisher<Object> execute(OperationExecutor executor, ExecutionContextAdapter context) throws MuleException {
```

Comment:
There's no way I know or care about the actual type and I don't like raw types. The Object doesn't hurt at all